integration-cli/docker_cli_daemon_test.go

```diff
@@ -2070,3 +2070,20 @@ func (s *DockerDaemonSuite) TestRunLinksChanged(c *check.C) {
 	c.Assert(err, check.NotNil, check.Commentf(out))
 	c.Assert(out, check.Not(checker.Contains), "1 packets transmitted, 1 packets received")
 }
+
+func (s *DockerDaemonSuite) TestDaemonUngracefulShutdownCleanUpRootfsOfOverlay(c *check.C) {
```

Comment:
No, that test for any graphdriver. All daemons will be run with `--storage-driver=overlay` if `DOCKER_GRAPHDRIVER=overlay` is set. Your test will be run only in that case, too.